test/runtests.jl

```diff
@@ -5,3 +5,8 @@ for t in tests
     println("* running $fp")
     include(fp)
 end
+
+using JET
+@testset "Static analysis" begin
+    @test isempty(JET.get_reports(report_package("StatsFuns", target_modules=(StatsFuns,))))
+end
```

Comment:
I think this should be checked only on Julia >= 1.7 (see also the comment in the logs: https://github.com/JuliaStats/StatsFuns.jl/actions/runs/3436051370/jobs/5729130463#step:6:88):
```suggestion

if VERSION >= v"1.7"
    using JET
    @testset "Static analysis" begin
        @test isempty(JET.get_reports(report_package("StatsFuns", target_modules=(StatsFuns,))))
    end
end
```